examples/nlp/neural_machine_translation_with_keras_hub.py

```diff
@@ -87,7 +84,7 @@
     origin="http://storage.googleapis.com/download.tensorflow.org/data/spa-eng.zip",
     extract=True,
 )
-text_file = pathlib.Path(text_file).parent / "spa-eng" / "spa.txt"
+text_file = pathlib.Path(text_file).parent / "spa-eng_extracted" / "spa-eng" / "spa.txt"
```

Comment:
[Unrelated, somewhat]

So, `keras.utils.get_file` adds `_extracted` on its own in newer versions of Keras? Any reason why this was done in the first place?

Reply:
See https://github.com/keras-team/keras/commit/dcefb139863505d166dd1325066f329b3033d45a

I made a change which makes it a bit shorter.